tests/behaviour/steps/inspectcli.py

```diff
@@ -36,8 +36,8 @@ def step_impl(context, csv_file: str):
 
 @When("the Metadata File json-ld is loaded to a rdf graph")
 def step_impl(context):
-    metadata_processor = MetadataProcessor(context.csvw_metadata_json_path)
-    context.csvw_metadata_rdf_graph = metadata_processor.load_json_ld_to_rdflib_graph()
+    csvw_rdf_manager = CsvwRdfManager(context.csvw_metadata_json_path)
+    context.csvw_metadata_rdf_graph = csvw_rdf_manager.load_json_ld_to_rdflib_graph()
```

Comment:
Again this can replaced with `context.csvw_metadata_rdf_graph = csvw_rdf_manager.rdf_graph`, can't it?